python/pyarrow/__init__.py

```diff
@@ -72,6 +72,7 @@ def parse_git(root, **kwargs):
                          Array, Tensor,
                          array, chunked_array, record_batch, table,
                          SparseCOOTensor, SparseCSRMatrix, SparseCSCMatrix,
+                         SparseCSRMatrix, SparseCOOTensor, SparseCSFTensor,
```

Comment:
Did `SparseCSCMatrix` disappear?

Reply:
It was doubled actually. Bad merge. Fixed.